### PR TITLE
calendar service doc

### PR DESCRIPTION
**Pull request by yinan-liu**

### CODE REVIEW

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 1]
```diff
@@ -0,0 +1,87 @@
+Calendar service is another ProActive micro-service allowing to visualize and manage (edit and delete) all jobs and a filter planned jobs in a calendar client such as Outlook, Thunderbird Lightning, Mac Calendar, etc. Be aware that calendar service could be running only when the ProActive scheduler is already running.
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: all jobs and a filter planned jobs -> I don't understand the sentence
calendar service could be running -> should

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+==== Installation
+
+Python3 and Radicale are mandatory for calendar service's installation, since calendar services uses Radicale as its CalDav server, and Radicale is written in pure Python.
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: Radicale as its CalDav server -> I don't understand the sentence

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+For more information about Radicale, please refer to <http://radicale.org/>.
+
+===== Windows
+A portable Radicale programm is available to be used in Windows environment, which includes all libraries and dependencies in one package to make Radicale server running on Windows.
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: programm -> program

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    
+    1. Run %PROACTIVE_HOME%/tools/radicale/windows/RadicalePortable_1.1.1_English.paf.exe and select a directory where the Radicale server will be installed (Note down this directory which is going to be used in the next step). 
+    +
+    WARNING: Do not tick the checkbox "Run Radicale Portable" in the last installation, Radicale server will be run later after some configurations
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: run -> launched

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    +
+    [source]
+    ----
+    cd  %PROACTIVE_HOME%/bin/
+    sudo ./calendar-service.sh start
+    ----
+    +
+
+===== Mac OS
+Python3 has to be installed already on Mac OS. Then user can follow the same steps for Linux to setup and start the calendar service.
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: to be installed already -> to be already installed 
Then user -> Then the user

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+==== Start up Calendar service
+
+===== Windows
+Go to the \bin foler and run calendar-service.bat
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: foler -> folder

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+    +
+    [source]
+    ----
+    cd  %PROACTIVE_HOME%/bin/
+    calendar-service.bat
+    ----
+
+===== Linux
+Go to the \bin foler and run calendar-service.sh
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: foler -> folder

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    +
+    [source]
+    ----
+    cd  %PROACTIVE_HOME%/bin/
+    sudo ./calendar-service.sh start
+    ----
+TIP: `calendar-service.sh` can be used with option stop, restart, status as well.
+
+==== Calendar service configuration
+Calendar service's default configuration can be override by creating an application.properties file in $PROACTIVE_HOME/config/calendar-service/.
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: can be override -> overriden

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+-  `server.port` - Calendar service rest server port, default value is "4242"
+
+-  `pa.scheduler.rest.url` - ProActive scheduler rest url, which is used to subscribe to the scheduler's jobs events listener list. Default value is "http://localhost:8080/rest/"
+
+-  `pa.scheduler.admin.login` - ProActive scheduler admin login name, used to connect to scheduler, default value is "admin"
+
+-  `pa.scheduler.admin.password` - ProActive scheduler admin login password, default value is "admin"
+
+-  `pa.scheduler.home` - ProActive installed home path, default value is relative parent directory "..""
```
Comment (laurianed) [inline src/docs/admin/references/CalendarServiceReference.adoc]: ".."" -> ".."

File: src/docs/user/references/CalendarServiceReference.adoc [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
+    ----
+
+* `host-address` - ProActive server address
+* `user` - ProActive scheduler user login name
+
+=== Outlook
+
+1. Open IE browser and go to the Calendar service URL
+
+2. A popup window appears and asking the credential. Use the credential of the scheduler `user` account where the login name is the one used in the calendar service url
```
Comment (laurianed) [inline src/docs/user/references/CalendarServiceReference.adoc]: appears and asking -> asks

File: src/docs/user/references/CalendarServiceReference.adoc [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+
+3. IE browser is going to ask would you like to `Open` or `Save` the calendar from the server, select `Open`
```
Comment (laurianed) [inline src/docs/user/references/CalendarServiceReference.adoc]: ask would you like to `Open` or `Save` -> ask if you'd like to `Open` or `Save`

File: src/docs/admin/references/CalendarServiceReference.adoc [hunk 12]
```diff
@@ -0,0 +1,87 @@
+Calendar service is another ProActive micro-service allowing to visualize and manage (edit and delete) all jobs and planned jobs in a calendar client such as Outlook, Thunderbird Lightning, Mac Calendar, etc. Be aware that calendar service could be running only when the ProActive scheduler is already running.
+
```
Comment (fviale) [inline src/docs/admin/references/CalendarServiceReference.adoc]: can be started only when ...

**Pull request merged**